scala/scala.bzl

```diff
@@ -831,11 +807,12 @@ def scala_library_suite(name,
 scala_junit_test = rule(
   implementation=_scala_junit_test_impl,
   attrs= _implicit_deps + _common_attrs + {
-      "patterns": attr.string_list(default=["Test"]),
+      "prefixes": attr.string_list(default=[]),
+      "suffixes": attr.string_list(default=[]),
```

Comment:
Also thought about it (and noted in the PR comments) and thought about adding macros.
The reason I'd leave it as-is is so that the choices will be explicit. Even if we add the macros their names will suggest the prefixes/suffixes which are being used.
WDYT?